calendars/sunpy.yaml

```
@@ -0,0 +1,13 @@
+name: SunPy Community Calendar
+events:
+  - summary: Weekly SunPy Community Call
+    description: |
+      The weekly call for the SunPy community.
+      Agenda: https://demo.hedgedoc.org/GAEnxycXQcCQLrAFN7ie8A?edit
```

```suggestion
      Agenda: https://sunpy.org/agenda
```
We will need to redirect this on our website.